homeassistant/components/vera/__init__.py

```diff
@@ -51,27 +49,42 @@
     extra=vol.ALLOW_EXTRA,
 )
 
-VERA_COMPONENTS = [
-    "binary_sensor",
-    "sensor",
-    "light",
-    "switch",
-    "lock",
-    "climate",
-    "cover",
-    "scene",
-]
 
+def setup(hass: HomeAssistant, base_config: dict) -> bool:
```

Comment:
We need to change this to a coroutine and rename it to `async_setup`.

We can only use the async api from async context and vice versa. It's easier to have the main context async and schedule parts to the executor (sync). Although we don't seem to do any I/O in `setup` so it's an easy conversion.